plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/settings/CodeWhispererConfigurable.kt

```diff
@@ -98,17 +98,6 @@ class CodeWhispererConfigurable(private val project: Project) :
                 }.comment(message("aws.settings.codewhisperer.automatic_import_adder.tooltip"))
             }
 
-            row {
-                text(message("codewhisperer.inline.settings.tab_priority.prefix"))
-                val amazonq = message("codewhisperer.inline.settings.tab_priority.choice.amazonq")
-                val jetbrains = message("codewhisperer.inline.settings.tab_priority.choice.jetbrains")
-                comboBox(listOf(amazonq, jetbrains)).bindItem(
-                    { if (CodeWhispererSettings.getInstance().isQPrioritizedForTabAccept()) amazonq else jetbrains },
```

Comment:
double checking `CodeWhispererSettings.getInstance().isQPrioritizedForTabAccept()` is not used right

Reply:
fixing